Sources/LanguageServerProtocolJSONRPC/JSONRPCConnection.swift

```diff
@@ -303,7 +313,17 @@ public final class JSONRPCConnection {
     send(_rawData: dispatchData)
   }
 
-  func send(encoding: (JSONEncoder) throws -> Data) {
+  private func sendMessageSynchronously(_ messageData: Data) {
+    let header = "Content-Length: \(messageData.count)\r\n\r\n"
+    guard let headerData = header.data(using: .utf8) else {
+      log("error encoding header", level: .error)
```

Comment:
This seems like it ought to be a fatalError.  There is no reason this should ever happen.

Reply:
done